yarn-project/circuit-types/src/interfaces/aztec-node.ts

```diff
@@ -56,21 +54,27 @@ export interface AztecNode {
    * @returns The sibling path for the leaf index.
    */
   getNoteHashSiblingPath(
-    blockNumber: BlockNumber,
+    blockNumber: L2BlockNumber,
     leafIndex: bigint,
   ): Promise<SiblingPath<typeof NOTE_HASH_TREE_HEIGHT>>;
 
   /**
    * Returns the index and a sibling path for a leaf in the committed l1 to l2 data tree.
    * @param blockNumber - The block number at which to get the data.
    * @param l1ToL2Message - The l1ToL2Message to get the index / sibling path for.
-   * @throws If the message is not found.
-   * @returns A tuple of the index and the sibling path of the message.
+   * @returns A tuple of the index and the sibling path of the message  (undefined if not found).
    */
-  getL1ToL2MessageIndexAndSiblingPath(
-    blockNumber: BlockNumber,
+  getL1ToL2MessageMembershipWitness(
```

Comment:
Did this change because the oracle which calls this func is called getL1ToL2MessageMembershipWitness as well so it makes sense to have it consistent.